.github/workflows/tomcat-common-ci.yml

```
@@ -6,7 +6,7 @@ on:
   schedule:
     - cron: '0 8 * * *' # everyday at 8am
   push:
-    branches: ["main", "develop"]
+    branches: ["main", "develop", "feature/addProxyChains"]
```

This was only needed for development -- remove

```suggestion
    branches: ["main", "develop"]
```

---

Removed it